CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## Unreleased
+### Added
+- Enhance date selection on reports
```

Comment:
Can we make this:

- Default start and end date on reports which require date selection
- Button on reports with date selection for the entire program
- Button on reports with date selection for the last week